tfx/types/artifact_test.py

```diff
@@ -164,6 +157,10 @@ def decode(self, value: bytes):
 
 class ArtifactTest(tf.test.TestCase):
 
+  def tearDown(self):
+    # This cleans up __subclasses__() that has InvalidAnnotation artifact classes.
+    gc.collect()
+
```

Comment:
This appears to have worked since the tests pass. 

In the future, we can look at using [pytest yield fixtures](https://docs.pytest.org/en/stable/how-to/fixtures.html#yield-fixtures-recommended) instead of setup and teardown.